proposals/16:upgrade-8/use.sh

```diff
@@ -5,6 +5,8 @@ set -e
 
 source /usr/src/upgrade-test-scripts/env_setup.sh
 
+yarn ava keys.test.js
```

Comment:
is this just debugging? tests don't belong in `use.sh`. the ones here are tech debt

Reply:
Yeah I noticed `keys.test.js` was not run and found an example of running it in `use.sh`

I've since moved it to `test.sh`